utils/utils.go

```diff
@@ -154,7 +152,7 @@ func AccAddressFromBech32(address string, bech32prefix string) (addr AccAddress,
 	return AccAddress(bz), nil
 }
 
-//==============================  AIRDROP UTILS  ================================
+// ==============================  AIRDROP UTILS  ================================
```

Comment:
@hoangdv2429 plz don't edit this

Reply:
if it is happening over and over, it could be an issue with his editor config.

Reply:
@ThanhNhann @faddat I'm really sorry about this. It's indeed related to me editor config, after cltr + s. My editor auto add space to some places (the one above for example). I'll try to fix my editor. I'm really sorry.